corehq/apps/translations/app_translations/download.py

```diff
@@ -19,39 +19,60 @@
     get_module_sheet_name,
     get_modules_and_forms_row,
 )
+from corehq.apps.translations.generators import EligibleForTransifexChecker
 
 
-def get_bulk_app_single_sheet_by_name(app, lang):
+def get_bulk_app_single_sheet_by_name(app, lang, eligible_for_transifex_only=False):
+    checker = EligibleForTransifexChecker(app)
+
     rows = []
     for module in app.modules:
+        if eligible_for_transifex_only and checker.exclude_module(module):
+            continue
         sheet_name = get_module_sheet_name(module)
         rows.append(get_name_menu_media_row(module, sheet_name, lang))
         for module_row in get_module_rows([lang], module):
+            if eligible_for_transifex_only:
+                field_name, field_type, translation = module_row
+                if checker.is_blacklisted(module.unique_id, field_type, field_name, [translation]):
+                    continue
             rows.append(get_list_detail_case_property_row(module_row, sheet_name))
 
         for form in module.get_forms():
+            if eligible_for_transifex_only and checker.exclude_form(form):
+                continue
             sheet_name = get_form_sheet_name(form)
             rows.append(get_name_menu_media_row(form, sheet_name, lang))
             for label_name_media in get_form_question_label_name_media([lang], form):
+                if (
+                    eligible_for_transifex_only and
+                    checker.is_label_to_skip(form.unique_id, label_name_media[0])
+                ):
+                    continue
                 rows.append(get_question_row(label_name_media, sheet_name))
 
     return OrderedDict({SINGLE_SHEET_NAME: rows})
 
 
-def get_bulk_app_sheets_by_name(app, exclude_module=None, exclude_form=None):
+def get_bulk_app_sheets_by_name(app, eligible_for_transifex_only=False):
     """
     Data rows for bulk app translation download
 
-    exclude_module and exclude_form are functions that take in one argument
+    `exclude_module` and `exclude_form` are functions that take in one argument
```

Comment:
this block should be removed now?

Reply:
(I love how Github lets you do the edits you missed :) )